modin/pandas/concat.py

```diff
@@ -132,3 +139,25 @@ def concat(
         else:
             result_df.columns = new_idx
     return result_df
+
+
+def _determine_name(objs: list, axis):
+    axis_getter = (lambda df: df.columns) if axis else (lambda df: df.index)
```

Comment:
You don't need this, you can use code below.

Reply:
answered in a comment below